mbed_host_tests/host_tests_runner/host_test_default.py

```diff
@@ -185,6 +189,12 @@ def callback__notify_prn(key, value, timestamp):
                             self.logger.prn_inf("%s(%s)"% (key, str(value)))
                             result = value
                             break
+                        elif key == '__reset_dut':
+                            # Disconnecting and re-connecting comm process will reset DUT
+                            dut_event_queue.put(('__host_test_finished', True, time()))
+                            p.join()
+                            self.mbed.update_device_info()
+                            p = start_conn_process()
```

Comment:
Good question. Actually this call has to be removed. As software reset does not require it.